samples/go/noms-merge/main.go

```diff
@@ -91,6 +98,60 @@ func nomsMerge() error {
 	}))
 }
 
+func resolve(aChange, bChange types.ValueChanged, a, b types.Value, path types.Path) (change types.ValueChanged, merged types.Value, ok bool) {
```

Comment:
This needs to be tested. You may want to drive the majority of the merge logic outside of the stdio loop, like pass the choices over a channel.